test/JDBC/input/alter/alter-function-vu-verify.sql

```diff
@@ -131,15 +131,62 @@ as
     )
 go
 
--- Test Case 9: Expect error for attempting to alter multi statement tvf
--- Alter Func Multi-statement tvf support will be added after BABEL-5149 is resolved
+-- Test Case 9: Alter multi statement tvf
 alter function alter_func_f5()
 returns @result TABLE(Id int) as begin
 insert into @result values (2)
 return
 end
 go
 
+select Id from alter_func_f5()
```

Comment:
Should add a test case about : 
[ schema_name. ] function_name   

With the schema_name